.github/workflows/publish-lib.yml

```
@@ -61,45 +52,19 @@ jobs:
         working-directory: lib
         run: pnpm build
 
-      - name: Set Version and Package Name (Tag - npmjs)
+      - name: Set Version (Tag - npmjs)
         if: ${{ startsWith(github.ref, 'refs/tags/v') }}
         working-directory: lib
         run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:3:13: Double quote to prevent globbing and word splitting

(shellcheck)